common/src/java/org/apache/hadoop/hive/conf/Constants.java

```diff
@@ -105,4 +105,6 @@ public class Constants {
 
   public static final String HTTP_HEADER_REQUEST_TRACK = "Request-Track";
   public static final String TIME_POSTFIX_REQUEST_TRACK = "_TIME";
+
+  public static final String IS_EXPLAIN = "explain";
```

Comment:
maybe `IS_EXPLAIN_PLAN` to be more specific?

Reply:
Renamed